modules/network/private-dns-zone/README.md

```diff
@@ -0,0 +1,84 @@
+# Private DNS Zone
+
+A bicep module for simplified deployment for Private DNS Zones and available configuration options.
+
+## Description
+
+Azure Private DNS provides a reliable, secure DNS service to manage and resolve domain names in a virtual network without the need to add a custom DNS solution. By using private DNS zones, you can use your own custom domain names rather than the Azure-provided names available today.
+
+## Parameters
+
+| Name                  | Type     | Required | Description                                                              |
+| :-------------------- | :------: | :------: | :----------------------------------------------------------------------- |
+| `name`                | `string` | Yes      | Required. Name of privateDNSZone, but be in DNS format like example.test |
+| `tags`                | `object` | No       | Optional. Tags for the resources.                                        |
+| `location`            | `string` | Yes      | Required. The location of the PrivateDNSZone. Should be global.          |
+| `virtualNetworkLinks` | `array`  | No       | Optional. Adding virtual network links to the PrivateDNSZone.            |
+| `aRecords`            | `array`  | No       | Optional. Specifies the "A" Records array                                |
+| `cnameRecords`        | `array`  | No       | Optional. Specifies the "CNAME" Records array                            |
+
+## Outputs
+
+| Name   | Type     | Description             |
+| :----- | :------: | :---------------------- |
+| `id`   | `string` | Get privateDnsZone ID   |
+| `name` | `string` | Get privateDnsZone name |
+
+## Examples
+
+### Example 1
+
+```
+module dnsZone 'br/public:network/private-dns-zone:0.1.0' = {
```

Comment:
```suggestion
module dnsZone 'br/public:network/private-dns-zone:1.0.1' = {
```
Looks like this was changed to `1.0.1` which is correct but got changed to `0.1.0` later?

Reply:
Now it is correct 0.0.1

Reply:
Sorry, I mean we should change it to `1.0.1`. For any new module, the version number for the examples should be whatever the version number defined in `version.json` (usually `1.0`) + `.1`. In this case the major and minor version in `version.json` is `1.0`, so the full version should be `1.0.1` once the PR is merged.

Reply:
Can you please review now